front/src/hooks/useQrResolver.ts

```diff
@@ -13,18 +13,20 @@ export enum IQrResolverResultKind {
   NOT_ASSIGNED_TO_BOX = "notAssignedToBox",
```

Comment:
I'd suggest to rework these cases:
- NO_BOXTRIBUTE_QR: `data.qrCode.__typename === "ResourceDoesNotExistError"  --> "No box found for this QR code!" toast (Currently line 65)
- NOT_AUTHORIZED_FOR_QR: `data.qrCode.__typename === "InsufficientPermissionError"  --> "You don't have permission to access this box!" toast (Currently line 56)
- NOT_ASSIGNED_TO_BOX: !data?.qrCode?.box --> forwarding to createBox component (Currently line 82)
- NOT_AUTHORIZED_FOR_BOX: data.qrCode.box?.__typename === "InsufficientPermissionError" --> "You don't have permission to access this box!" toast
- NOT_AUTHORIZED_FOR_BASE: data.qrCode.box?.__typename === "UnauthorizedForBaseError" --> show the name of the other base
- SUCCESS --> forwarding to Boxview BoxView
- FAIL --> fallback case for any unexpected error